lib/blockchain/hardwareWallet.js

```diff
@@ -0,0 +1,305 @@
+const assert = require('bsert');
+const blgr = require('blgr');
+const { Lock } = require('bmutex');
+const { HARDWARE_WALLET_TIMEOUT, LOG_LEVEL } = require('../constants');
+
+// conditionally import based on environment
+let bledger, Device;
+
+if (process.env.BROWSER) {
```

Comment:
This makes me think we should come up with a special browser entrypoint for bpanel-utils like the rest of the bcoin family of libraries (for when this gets moved to bpanel-utils). I think it makes sense to assume that most plugins like `bwallet` will be solely browser, which means we can stick to supporting things like dynamic imports (which allows for better code splitting, caching, hot module replace, etc.) but the utilities for example can be used in both environments.